examples/pgm_conversion.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model IO project <dynamic.grid.calculation@alliander.com>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+import logging
+from pathlib import Path
+
+import pandas as pd
+import structlog
+from power_grid_model import PowerGridModel
+
+from power_grid_model_io.converters.pgm_converter import PgmConverter
+
+DATA_DIR = Path(__file__).parent / "data"
+
+if __name__ == "__main__":
+    structlog.configure(wrapper_class=structlog.make_filtering_bound_logger(logging.DEBUG))
+
+    input_file = DATA_DIR / "1os2msr" / "input.json"
```

Comment:
I think we should rename these examples

Reply:
1os2msr doesn't mean anything in English

Reply:
Done, I've called it tiny-net.